molgenis-metadata-manager/src/main/frontend/src/components/MetadataManagerEntityEditForm.vue

```diff
@@ -0,0 +1,222 @@
+          <div class="col">
+            <multiselect v-model="lookupAttributes" :options="attributes" label="label"
+                         selectLabel="" deselectLabel="" placeholder="Select attributes" multiple></multiselect>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style src="vue-multiselect/dist/vue-multiselect.min.css"></style>
+<style lang="scss">
+  @import "~variables";
+  @import "~mixins";
+
+  .checkbox-column {
+    padding-top: 11px
+  }
+
+  /*screen-md border on inner column when columns aligned next to each other*/
+  @include media-breakpoint-up(md) {
+    .col-md-4.inner-column {
+      border-right: solid black thin;
+    }
+  }
+
+  .multiselect__tag {
+    background-color: darken($red, 20%);
+  }
+</style>
+
+<script>
+  import { mapState, mapGetters } from 'vuex'
+  import { UPDATE_EDITOR_ENTITY_TYPE } from '../store/mutations'
+  import { SAVE_EDITOR_ENTITY_TYPE, DELETE_ENTITY_TYPE } from '../store/actions'
+  import { getConfirmBeforeDeletingProperties } from '../store/getters'
+
+  import Multiselect from 'vue-multiselect'
+  import SaveButton from './generic-components/SaveButton'
+
+  export default {
+    name: 'metadata-manager-entity-edit-form',
+    methods: {
+      saveEntityType () {
+        this.$store.dispatch(SAVE_EDITOR_ENTITY_TYPE)
+      },
+      deleteEntityType (selectedEntityTypeId) {
+        this.$swal(getConfirmBeforeDeletingProperties(selectedEntityTypeId)).then(() => {
+          this.$router.push({ path: '/' })
+          this.$store.dispatch(DELETE_ENTITY_TYPE, selectedEntityTypeId)
+        }).catch(this.$swal.noop)
+      }
+    },
+    computed: {
+      ...mapState(['editorEntityType', 'packages']),
+      ...mapGetters({
+        abstractEntities: 'getAbstractEntities',
+        attributes: 'getEditorEntityTypeAttributes',
+        isEntityTypeEdited: 'getEditorEntityTypeHasBeenEdited'
+      }),
+      entityTypeParent: {
+        get () {
+          return this.$store.state.editorEntityType.entityTypeParent
+        },
+        set (value) {
+          this.$store.commit(UPDATE_EDITOR_ENTITY_TYPE, { key: 'entityTypeParent', value: value })
+        }
+      },
+      abstract0: {
+        get () {
+          return this.$store.state.editorEntityType.abstract0
+        },
+        set (value) {
+          this.$store.commit(UPDATE_EDITOR_ENTITY_TYPE, { key: 'abstract0', value: value })
+        }
+      },
+      label: {
+        get () {
+          return this.$store.state.editorEntityType.label
+        },
+        set (value) {
+          this.$store.commit(UPDATE_EDITOR_ENTITY_TYPE, { key: 'label', value: value })
+        }
+      },
+      description: {
+        get () {
+          return this.$store.state.editorEntityType.description
+        },
+        set (value) {
+          this.$store.commit(UPDATE_EDITOR_ENTITY_TYPE, { key: 'description', value: value })
+        }
+      },
+      package0: {
```

Comment:
This is the model as it is coming from the server, didn't want to change it due to collision errors on save